src/main/java/frc/robot/Telemetry.java

```diff
@@ -88,8 +88,8 @@ public void telemeterize(SwerveDriveState state) {
                 pose.getRotation().getDegrees()
         });
 
-        //Logger.recordOutput("pose/Pose2d", pose);
-        //Logger.recordOutput("pose/fieldPose", table.getDoubleArrayTopic("/Pose/robotPose"));
+        Logger.recordOutput("pose/Pose2d", pose);
```

Comment:
Worth noting: we are telemeterizing the rotation in degrees - can we get this changed to radians to match our standard units

Reply:
Nevermind, I just am being silly

Reply:
Wait, no, we are doing telemetry in degrees. Can we do it in radians? 

The reason I am convinced of this is because the robot is undrivable in AdvantageScope without changing to degrees, and it looks like the pose[2] is in degrees when rotating